examples/demo_dlmbl/solution.py

```diff
@@ -491,11 +566,9 @@ def log_batch_jupyter(batch):
 </div>
 """
 
-
 # %%
-
-GPU_ID = 0
-n_samples = len(phase2fluor_data.train_dataset)
+GPU_ID = 1
```

Comment:
Should probably set this programatically.

Reply:
do mean this or using the VSTrainer to allocate it itself by providing `devices=1`?

Reply:
I think if there is only 1 GPU on the node, `GPU_ID=0` is equivalent to `devices="auto"`. However `GPU_ID=1` might not work since there isn't a second one.